innofw/core/models/torch/lightning_modules/segmentation.py

```diff
@@ -136,6 +136,7 @@ def stage_step(self, stage, batch, do_logging=False, *args, **kwargs):
 
         if stage in ["train", "val"]:
             loss = self.log_losses(stage, predictions, label)
+            self.log("loss", loss)
```

Comment:
not needed. As function above does the same but also adds stage prefix

Reply:
fixed